2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -32,6 +32,10 @@ object Dependencies {
     val slf4j                = "1.7.26"
     val config               = "1.3.4"
     val prometheus           = "0.6.0"
+    /** TODO Make sure cbor version stays aligned with jackson-core
+     *  See snowplow/snowplow/issues/4266
+     */
+    val cbor                 = "2.9.10"
     // Scala
     val collectorPayload     = "0.0.0"
     val scalaz7              = "7.0.9"
```

Comment:
How do we still have `scalaz7` dependency, @benjben?

Reply:
It got removed only from SCE, not SSC

Reply:
We definitely should do it in 1.2.0.

Reply:
Definitely!